tests/ffi/qm_nested/test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+# Declare global variables and initialize to 0
+NESTED_NAME=0
+IS_NESTED_RUNNING=0
+NESTED_STATUS=0
+
+# Create nested container fedora inside the QM.
+# Function to create and start nested container fedora, while executing podman commands from inside the QM container.
+# Extract nested container's valuable information and store it inside the global variables for later use.
+# Globals:
+#  NESTED_NAME: The nested container name in the QM container.
+#  NESTED_STATUS: The nested container status in the QM container.
+#  IS_NESTED_RUNNING: The nested container is running true or false value.
+create_nested() {
+        podman exec -it qm bash -c "podman run -d -it --name fedora fedora /bin/bash"
+        echo "Nested container got started."
+        NESTED_NAME=$(podman exec -it qm bash -c "podman inspect fedora --format '{{.Name}}'")
+        echo "Nested container name is: $NESTED_NAME"
+        IS_NESTED_RUNNING=$(podman exec -it qm bash -c "podman inspect --format='{{.State.Running}}' fedora")
+        echo "Is nested container $NESTED_NAME is running? $IS_NESTED_RUNNING"
+        NESTED_STATUS=$(podman exec -it qm bash -c "podman inspect --format='{{.State.Status}}' fedora")
+        echo "Nested container $NESTED_NAME status is: $NESTED_STATUS"
+}
+
+# Function to check and verify that nested container fedora was created and is running inside the QM container.
+# Globals:
+#  NESTED_NAME: The nested container name in the QM container.
+#  NESTED_STATUS: The nested container status in the QM container.
+#  IS_NESTED_RUNNING: The nested container is running true or false value.
+compare_values() {
+        local expected_nested_name="fedora"
+        local expected_nested_status="running"
+
+        if [ "$NESTED_NAME" == 0 ] || [ "$IS_NESTED_RUNNING" == 0 ] || [ "$NESTED_STATUS" == 0 ]; then
+                echo "FAIL: One or multiple nested container's values are 0. Ensure the nested container is running on the QM container and the environment configured correctly."
```

Comment:
Please replace "multiple nested" and nested at all, 
With similar to this
Failed to run container inside QM

It could cause misleading understanding,
QM image is a file system partition